05-JWT-Basics/starter/errors/bad-request.js

```diff
@@ -0,0 +1,10 @@
+const CustomAPIError = require('./custom-error')
+const { StatusCodes } = require('http-status-codes');
+class BadReqest extends CustomAPIError {
```

Comment:
```suggestion
class BadRequest extends CustomAPIError {
```
Typo here, no big deal though